backend/dynamic_parameter_service/dynamic_parameter_service.go

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
+
+// +build ui
+
+package dynamic_parameter_service
+
+import (
+	"fmt"
+    "io/ioutil"
+    "errors"
+    "os"
+
+    "github.com/gin-gonic/gin"
+    "github.com/omec-project/webconsole/backend/logger"
+)
+
+type route struct {
+    Pattern string
+	EnvVariable string
+}
+
+func AddDynamicParameterService(engine *gin.Engine) {
+	logger.WebUILog.Infoln("Adding dynamic parameters service")
+    group := engine.Group("/config/parameter")
+    for _, parameter := range dynamicParameters {
+        serveDynamicParameter(group, parameter.Pattern, parameter.EnvVariable);
+	}
+}
+
+func serveDynamicParameter(group *gin.RouterGroup, pattern string, envVariable string){
+    fileContent, err := readFileFromEnvVariable(envVariable)
+	if err != nil {
+        logger.WebUILog.Infof("/config/parameter%s route will not be served", pattern)
+		return
+	}
+	group.GET(pattern, func(c *gin.Context) {
```

Comment:
I'm thinking that maybe we should get rid of those json files and use the db instead, alongisde a POST/DELETE call.

Our charm would use this API call to add/remove gnb's and upf's.